kubernetes-model-generator/kubernetes-model-certificates/src/generated/java/io/fabric8/kubernetes/api/model/certificates/v1beta1/CertificateSigningRequestSpec.java

```diff
@@ -63,8 +62,6 @@
 public class CertificateSigningRequestSpec implements Editable<CertificateSigningRequestSpecBuilder> , KubernetesResource
 {
 
-    @JsonProperty("expirationSeconds")
-    private Integer expirationSeconds;
```

Comment:
I guess this is due to same reason as https://github.com/fabric8io/kubernetes-client/pull/6174#discussion_r1688020365

Reply:
A test is using this field 

https://github.com/fabric8io/kubernetes-client/blob/02a94099e07f25dc30a906631261074935e2b47f/kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CertificateSigningRequestTest.java#L130

Reply:
> I guess this is due to same reason as https://github.com/fabric8io/kubernetes-client/pull/6174#discussion_r1688020365

Exactly, yes. Users should have already transitioned to v1 or should not be using that field (unless I'm missing something)

> A test is using this field

Changed, thx!